### PR TITLE
Rule8810

### LINKED ISSUE #48: Rule 8810 : A CIN case cannot have a Reason for Closure without a CIN Closure Date
If \<ReasonForClosure\> (N00103) is present then \<CINclosureDate\> (N00102) must also be present

### PR DESCRIPTION
**Pull request by willLPD2I**
closes #48

### CODE REVIEW

**Comment by willLPD2I** [comment]
I've updated the link_id to a non null, hope it's right now!

File: cin_validator/rules/cin2022_23/rule_8810.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+
+def test_validate():
+    # Create some sample data such that some values pass the validation and some fail.
+    fake_data_frame = pd.DataFrame(
+        [
+            {
+                "LAchildID": "child1",
+                "CINdetailsID": "CINID1",
+                "ReasonForClosure": "26/05/2000",
```
Comment (tab1tha) [inline cin_validator/rules/cin2022_23/rule_8810.py:75]: This does work. However, the data values for ReasonForClosure are more relatable if they are strings. The reason cannot be a date. Date columns tend to have the word "date" in their name.

Comment (tab1tha) [reply cin_validator/rules/cin2022_23/rule_8810.py:75]: You can use random strings like "aaa", "bbb", "abc" ....

File: cin_validator/rules/cin2022_23/rule_8810.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+    # SUBMIT ERRORS
+    # Generate a unique ID for each instance of an error. In this case,
+    # - If only LAchildID is used as an identifier, multiple instances of the error on a child will be understood as 1 instance.
+    # We don't want that because in reality, a child can have multiple instances of an error.
+    # - If we use the LAchildID-CPPstartDate combination, that artificially cancels out the instances where a start date repeats for the same child.
+    # Another rule checks for that condition. Not this one.
+    # - It is very unlikely that a combination of LAchildID-CPPstartDate-CPPendDate will repeat in the DataFrame.
+    # Hence, it can be used as a unique identifier of the row.
+
```
Comment (tab1tha) [inline cin_validator/rules/cin2022_23/rule_8810.py:55]: You can remove this comment block. This explanation is only relevant for the first rule that it explained.

**Review by tab1tha — CHANGES_REQUESTED** [review]
After the changes suggested, everything else is good to go.

Nice work !

File: cin_validator/rules/cin2022_23/rule_8810.py (hunk 1 above)
Comment (willLPD2I) [reply cin_validator/rules/cin2022_23/rule_8810.py:75]: HAha, I was just trying to save time! I'll do this now

Comment (willLPD2I) [reply cin_validator/rules/cin2022_23/rule_8810.py]: done

**Review by tab1tha — APPROVED** [review]

**Pull request merged**